vt-digital-libraries-platform/vt-digital-libraries-platform.github.io
# Setup website skeletal structure

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2046) (:star:)

# What does this Pull Request do? (:star:)
Setup website skeletal structure. See screenshot below

# How should this be tested?
1. clone this repo
```
bundle
```

2. Start server
```
bundle exec jekyll build
bundle exec jekyll serve
```
3. Check local server url: http://127.0.0.1:4000

# Interested parties
Tag (@andreaWaldren @whunter @tingtingjh @jimtuttle ) interested parties
<img width="1289" alt="Screen Shot 2020-02-21 at 8 17 45 AM" src="https://user-images.githubusercontent.com/7999195/75037652-ab9e9580-5482-11ea-9706-eb443210fcd0.png">

(:star:) Required fields
